### ISSUE TITLE
Add support for Runtime backends

### ISSUE THREAD

**Issue by kevinsung** [Qiskit-Partners/mapomatic#50]
Runtime backends have a different API than IBMQ backends. Example:
```python
import mapomatic
from qiskit import QuantumCircuit
from qiskit_ibm_runtime import QiskitRuntimeService

qc = QuantumCircuit(5)
qc.h(0)
qc.cx(0, 1)
qc.measure_all()

service = QiskitRuntimeService()
backend = service.backend("ibm_auckland")
layout = mapomatic.best_overall_layout(qc, [backend])
```
```
Traceback (most recent call last):
  File "/home/kjs/projects/scratch/mapomatic-update.py", line 12, in <module>
    layout = mapomatic.best_overall_layout(qc, [backend])
  File "/home/kjs/projects/mapomatic/mapomatic/layouts.py", line 203, in best_overall_layout
    best_out.append((layout, backend.name(), error))
TypeError: 'str' object is not callable
```

**Comment by nonhermitian** [comment]
Yes, this is an unfortunate breaking change in Runtime

**Comment by nonhermitian** [comment]
Should be closed in #62
